python/ql/lib/semmle/python/frameworks/Stdlib.qll

```diff
@@ -2683,14 +2698,12 @@ private module StdlibPrivate {
         method = "log" and
         msgIndex = 1
       |
-        this = Logger::instance().getMember(method).getACall()
-        or
-        this = API::moduleImport("logging").getMember(method).getACall()
```

Comment:
I don't agree about this removal. I see that `API::moduleImport("logging")` is now modeled as a LoggerInstance... but that is not true, these are helper methods defined on the module leve.

Reply:
Is there value in modelling instances specifically? Or could we just rename `instance` to something like `loggingMethodProvider`?

Reply:
Reverted to previous modelling approach as discussed offline.